src/util.js

```diff
@@ -8,3 +10,9 @@ export function extend(obj, props) {
 }
 
 
+/** Coerces and returns a lowercased representation of the argument.
+ *  @param {mixed} str
+ */
+export function lowercase (str) {
```

Comment:
I can see for perf. Size doesn't make sense because method accesses aren't minified, but function names are. I was trying to offset the amount of bytes added with this. I'll check later when I have a chance.